integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_sex_gender_page.cy.ts

```diff
@@ -0,0 +1,68 @@
+//  Feature: Referrer completes 'Sex and Gender identity' question page
+//    So that I can complete the 'Equality questions' task
+//    As a referrer
+//    I want to answer questions on the sex and gender page
+//
+//  Scenario: submit sex and gender itentity answers
+//    Given I'm on the 'Sex and gender identity' task page
+//    When I submit 'female' as their sex, and 'No' on the gender identity question
```

Comment:
perhaps it would be more maintainable to make this action more generic, e.g.

> "When I give valid answers to the 'Sex and gender identity' questions"

and leave the specifics of the questions and answers to the `SexAndGenderPage` helper's `selectSex()` and `confirmGenderIdentity()` functions?

Reply:
yep good point!